src/PublicApiGenerator/CSharpAlias.cs

```diff
@@ -0,0 +1,42 @@
+namespace PublicApiGenerator
+{
+    internal static class CSharpAlias
```

Comment:
I'm a fan of this change. 'Alias' means something specific in C#, so I'd call it `CSharpTypeKeyword` or something.

Reply:
Maybe